### PR TITLE
Grouping parentheses instead of capturing parentheses

### PR DESCRIPTION
**Pull request by rebeka-catalina**
As already mentioned in the README file, the usage of of capturing parathese produces errors, so it's important to use grouping parantheses there instead.

### CODE REVIEW

File: nsfw/README [hunk 1]
```diff
@@ -47,7 +47,25 @@ done with a single regex, which matches all of them:
 Another use case could be, that you are simply not
 interested in postings about christmas.
 
-	/christmas([-_ ]?(tree|time|eve|pudding))?/
+	/christmas([-_ ]?(?:tree|time|eve|pudding))?/
```
Comment (MrPetovan) [inline nsfw/README]: What about the first set of parentheses, shouldn't they be grouping as well?

**Comment by rebeka-catalina** [comment]
Yes, you're right.

**Comment by rebeka-catalina** [comment]
This is not my day -.-
Sorry

**Comment by tobiasd** [comment]
> This is not my day -.-

Why, it is just gamification of useless commit statistics ;-)

**Pull request merged**